### DISCUSSION TITLE
In model PageContent flag in_navigation should be configurable

### DISCUSSION THREAD

**Discussion by jrief** [django-cms/django-cms#7802] [General]
In our setup, newly created pages do not show up in the navigation by default. I believe that such a configuration might also be useful for others.

For this I had to change the default settings of that model and add another configuration directive. Shall we make this configurable be default? I then would create a pull request for it.

**Comment by marksweb** [comment]
Yes that seems fair to configure.

It makes me wonder about this CMS config that we have in v4 again. I think most of us still don't have an understanding of this side of things. I gather it's about how apps work together, but this might tie into it.
